precipitation/ngsild-payloads/withdrawal.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "urn:ngsi-ld:WaterWithdrawal:OPR0000144308",
+    "type": "WaterWithdrawal",
+    "withdrawal": {
+      "type": "Property",
+      "value": 0.0,
+      "observedAt": "2020-01-01T00:00:00Z",
+      "relationship": {
```

Comment:
this is not what I meant.

I meant you could have a relationship from the `WaterWithdrawal` entity to the `Structure` entity, at the entity level, and not at each property level, as I guess a `WaterWithdrawal` entity is always linked to one and only one `Structure` entity (and don't name it `relationship` as this will probably brings some trouble).